benchmark/src/main/scala/keycloak/scenario/KeycloakScenarioBuilder.scala

```diff
@@ -603,6 +619,22 @@ class KeycloakScenarioBuilder {
     this
   }
 
+  def getUserUUID(): KeycloakScenarioBuilder = {
+    chainBuilder = chainBuilder
+      .exec(http("List Users and get a User UUID")
+        .get(ADMIN_ENDPOINT + "/users")
+        .header("Authorization", "Bearer ${token}")
+        .queryParam("first", 0)
+        .queryParam("max", 50)
+        .check(
+          status.is(200))
+        //ToDo Randomize the index of the user object to not fetch skewed Session stats
+        .check(jsonPath("$[20].id").notNull.saveAs("userUUID"))
```

Comment:
As with the example above, this is a suggestion to clear the TODO. If the URL doesn't work as expected, I'd be happy to pair.

```suggestion
        .get(ADMIN_ENDPOINT + "/users?username=${username}&exact=true")
        .header("Authorization", "Bearer ${token}")
        .queryParam("first", 0)
        .queryParam("max", 1)
        .check(
          status.is(200))
        .check(jsonPath("$[0].id").notNull.saveAs("userUUID"))
```